### ISSUE TITLE
Add possibility to turn Off Cilium Socket-Level Load Balancing 

### ISSUE THREAD

**Issue by 4FunAndProfit** [vitobotta/hetzner-k3s#490]
Hello ! 

I want to use this beautiful tool for my project!

Yet I have an issue, is it possible to add possibility to turn off Off Socket-Level Load Balancing in cilium config?

Indeed I need to use with Linkerd and this is required (https://linkerd.io/2-edge/reference/cluster-configuration/#cilium)
```
This behavior can be turned off in Cilium by [turning off socket-level load balancing for pods](https://docs.cilium.io/en/v1.13/network/istio/#setup-cilium) through the CLI option --config bpf-lb-sock-hostns-only=true, or through the Helm value socketLB.hostNamespaceOnly=true.
```
I don't see any way to add this config to cni yaml config ?

Thanks a lot for your help :))

**Comment by 4FunAndProfit** [comment]
Oh Thanks a lot @vitobotta ! 🥰

**Comment by vitobotta** [comment]
Np :)

**Comment by vitobotta** [comment]
@KristijanL Yes, that makes a lot of sense. It is easier to give the path of the custom values file instead of trying to copy all the chart settings in code. That would be one more thing to keep updated. I will check if I can do this for the next release!
